packages/templates/src/raw/kitchen-stories/template.json

```diff
@@ -0,0 +1,6278 @@
+            "id": 1794,
+            "title": "page01_logo",
+            "alt": "page01_logo",
+            "local": false,
+            "baseColor": [255, 255, 255]
+          },
+          "type": "image",
+          "x": 130.5,
+          "y": 0,
+          "width": 151,
+          "height": 81,
+          "mask": {
+            "type": "rectangle",
+            "showInLibrary": true,
+            "name": "Rectangle",
+            "path": "M 0,0 1,0 1,1 0,1 0,0 Z",
+            "ratio": 1,
+            "supportsBorder": true
+          },
+          "id": "7fef7ce3-34c1-4536-acd0-a875de006c01"
+        }
+      ],
+      "backgroundColor": {
+        "color": {
+          "r": 255,
+          "g": 255,
+          "b": 255
+        }
+      },
+      "type": "page",
+      "id": "dafd5c10-3f8b-4476-bd6a-6bb1620addd3",
+      "defaultBackgroundElement": {
+        "opacity": 100,
+        "flip": {
+          "vertical": false,
+          "horizontal": false
+        },
+        "rotationAngle": 0,
+        "lockAspectRatio": true,
+        "backgroundColor": {
+          "color": {
+            "r": 255,
+            "g": 255,
+            "b": 255
+          }
+        },
+        "x": 1,
+        "y": 1,
+        "width": 1,
+        "height": 1,
+        "mask": {
+          "type": "rectangle"
+        },
+        "isBackground": true,
+        "isDefaultBackground": true,
+        "type": "shape",
+        "id": "e0ad34a8-c0a4-4888-853b-b708485b61d4"
+      },
+      "animations": [
+        {
+          "id": "53be4bcb-6693-4e9c-8f9f-d8c1f2e065be",
+          "type": "effect-fade-in",
+          "targets": ["78e7271d-a523-4873-9481-aeb7eaa6da91"],
+          "duration": 1000,
+          "delay": 200
+        }
+      ],
+      "pageTemplateType": "cover"
+    },
+    {
+      "elements": [
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundColor": {
+            "color": {
+              "r": 255,
+              "g": 255,
+              "b": 255
+            }
+          },
+          "x": 48,
+          "y": 0,
+          "width": 304,
+          "height": 540,
+          "mask": {
+            "type": "rectangle"
+          },
+          "isBackground": true,
+          "type": "video",
+          "id": "0edc5b8b-a567-48c5-b988-e5b14d4cefc1",
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "resource": {
+            "type": "video",
+            "mimeType": "video/mp4",
+            "creationDate": "2021-05-10T12:24:20",
+            "src": "__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/kitchen-stories/page02_bg_video.mp4",
+            "width": 304,
+            "height": 540,
+            "poster": "__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/kitchen-stories/page02_bg_poster.jpg",
+            "posterId": 3515,
```

Comment:
This id is invalid id. Maybe this should just be 0?

Reply:
I think this is WP media ID, will check if the `posterId` is used if the `poster` is present and will make change accordingly.